chromeless/chromeless.py

```diff
@@ -11,7 +11,11 @@
 class Chromeless():
     REQUIRED_SERVER_VERSION = 2
 
-    def __init__(self, gateway_url=None, gateway_apikey=None, chrome_options=None, function_name='chromeless-server-prod'):
+    def __init__(self, gateway_url=None, gateway_apikey=None, chrome_options=None,
+                 function_name='chromeless-server-prod', boto3_session=None):
+        if boto3_session is None:
+            boto3_session = boto3
+        self.boto3_session = boto3_session
```

Comment:
```python
self.boto3_session = boto3_session if boto3_session is None else boto3
```

one line is also possible.

Reply:
Yeah, but won't it add an unnecessary `else` clause?

Reply:
Sorry, my comment was wrong, but commit seems ok for me?
https://github.com/faizanalhassan/pythonista-chromeless/blob/dev/boto-session/chromeless/chromeless.py#L14-L16

Reply:
Yeah, it's good also.

Reply:
Is there a way to talk to you in person? I am using your chromeless in a project, but need your suggestion.

Reply:
I can send my work email via twitter and you can invite me your schedule and use Google Meet, but it's always welcome to create a issue even if topic and problem is not clear or not related.

Reply:
I think creating another issue is better

Reply:
Alright sounds good to me. I just sent email actually tho, please forget.